Dockerfile.import

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:bionic
+FROM ubuntu:20.04
```

Comment:
Ubuntu 20.04 is even older than the 22.04 used in the main file. Why not something more recent?